OpenRA.Mods.Common/Traits/Health.cs

```diff
@@ -17,13 +19,39 @@ public class HealthInfo : ITraitInfo, UsesInit<HealthInit>
 	{
 		[Desc("HitPoints")]
 		public readonly int HP = 0;
-
-		[Desc("Physical size of the unit used for damage calculations. Impacts within this radius apply full damage.")]
-		public readonly WDist Radius = new WDist(426);
-
 		[Desc("Trigger interfaces such as AnnounceOnKill?")]
 		public readonly bool NotifyAppliedDamage = true;
 
+		[FieldLoader.LoadUsing("LoadShape")]
+		public readonly IHitShape Shape;
+
+		static object LoadShape(MiniYaml yaml)
+		{
+			IHitShape ret;
+
+			var shapeNode = yaml.Nodes.Find(n => n.Key == "Shape");
+			var shape = shapeNode != null ? shapeNode.Value.Value : string.Empty;
+
+			if (!string.IsNullOrEmpty(shape))
+			{
+				ret = Game.CreateObject<IHitShape>("HitShape" + shape);
```

Comment:
The idiom is to add a suffix instead of a prefix:
```
OpenRA.Game/GameRules/ActorInfo.cs:112:    var info = Game.CreateObject<ITraitInfo>(traitName + "Info");
OpenRA.Game/GameRules/WeaponInfo.cs:78:    var ret = Game.CreateObject<IProjectileInfo>(proj.Value + "Info");
OpenRA.Game/GameRules/WeaponInfo.cs:88:    var ret = Game.CreateObject<IWarhead>(node.Value.Value + "Warhead");
OpenRA.Game/Map/ActorReference.cs:46:      var info = Game.CreateObject<IActorInit>(traitName + "Init");
OpenRA.Game/Widgets/WidgetLoader.cs:77:    return Game.ModData.ObjectCreator.CreateObject<Widget>(widgetType + "Widget", args);

```
It also sounds better.